tests/unittests/detection/__init__.py

```diff
@@ -8,3 +8,7 @@
 _DETECTION_VAL = os.path.join(_PATH_ALL_TESTS, "_data", "detection", "instances_val2014_100.json")
 _DETECTION_BBOX = os.path.join(_PATH_ALL_TESTS, "_data", "detection", "instances_val2014_fakebbox100_results.json")
 _DETECTION_SEGM = os.path.join(_PATH_ALL_TESTS, "_data", "detection", "instances_val2014_fakesegm100_results.json")
+_DETECTION_OKS = os.path.join(
+    _PATH_ALL_TESTS, "_data", "detection", "person_keypoints_val2014_fakekeypoints100_results.json"
+)
+_DETECTION_OKS_VAL = os.path.join(_PATH_ALL_TESTS, "_data", "detection", "keypoint2014_val.json")
```

Comment:
is this just the default data from pycocotools?

Reply:
Don't know where I found *2014* anymore, but in any case `person_keypoints_val2017.json` is available in the trainval2017 zip on cocodataset.org so that would need to be bundled in that file the tests fetch.